scripts/deploy.js

```diff
@@ -0,0 +1,73 @@
+const fs = require("fs");
+const hre = require("hardhat");
+const { exit } = require("process");
+const { ethers } = hre;
+
+const network = process.env.HARDHAT_NETWORK;
+
+// Initialize object that will hold all deploy info. We'll continually update this and save it to
+// a file using the save() method below
+const parameters = {
+  admin: null,
```

Comment:
There's no admin right? I think this + `adminWallet` are just artifacts of our other deploy scripts. Just want to confirm

Reply:
Correct, no admin. That's just the deployer address being saved, though you're right, not great naming in this case.